tests/test_more.py

```diff
@@ -145,3 +145,54 @@ def test_dedup_handler(logger):
     assert 2 == len(test_handler.records)
     assert 'message repeated 2 times: foo' in test_handler.records[0].message
     assert 'message repeated 1 times: bar' in test_handler.records[1].message
+
+
+class TestRiemannHandler(object):
+
+    @require_module("riemann_client")
+    def test_happy_path(self, logger):
```

Comment:
I don't know if I understood correctly. You're addressing the problem that these tests are lacking integration tests, or some syntax makes Travis ignore my tests (without my knowledge)?